src/Symfony/Component/Serializer/Normalizer/BackedEnumNormalizer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * Normalizes a {@see \BackedEnum} enumeration to a string or an integer.
+ *
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+final class BackedEnumNormalizer implements NormalizerInterface, DenormalizerInterface, CacheableSupportsMethodInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return int|string
+     */
+    public function normalize($object, $format = null, array $context = [])
+    {
+        if (!$object instanceof \BackedEnum) {
+            throw new InvalidArgumentException('The data must belong to a backed enumeration.');
+        }
+
+        return $object->value;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        return interface_exists(\BackedEnum::class) && $data instanceof \BackedEnum;
```

Comment:
```suggestion
        return $data instanceof \BackedEnum;
```

It's okay to use `instanceof` with undefined classes/interfaces.